interfaces/connection.go

```diff
@@ -103,19 +103,31 @@ func getAttribute(snapName string, ifaceName string, staticAttrs map[string]inte
 }
 
 // NewConnectedSlot creates an object representing a connected slot.
-func NewConnectedSlot(slot *snap.SlotInfo, dynamicAttrs map[string]interface{}) *ConnectedSlot {
+func NewConnectedSlot(slot *snap.SlotInfo, staticAttrs, dynamicAttrs map[string]interface{}) *ConnectedSlot {
+	var static map[string]interface{}
+	if staticAttrs != nil {
```

Comment:
Will `staticAttrs` and `slot.Attrs` be, at the same time, non-nil and different?

Reply:
In theory they shouldn't be different. However since we decided to keep both dynamic and static attributes in conn state (for consistency, and to have full picture of all attributes at the time connection was made), this is setting the stage for the other PR where reloadConnection uses static attributes from conn state.